cves/CVE-2008-2939.yml

```
@@ -201,4 +210,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The mistake that was made was a coding mistake. The design or developers did
```

I am also left somewhat confused by this sentence, truth be told. I mean it's kind of nit-picky, but can a "design" really "not think to validate input"? I would probably say something like this
```suggestion
    The mistake that was made was a coding mistake. The designers and developers did
```